common/src/main/resources/assets/ping-wheel/lang/uk_ua.json

```diff
@@ -0,0 +1,44 @@
+{
+	"ping-wheel.key.ping-location": "Поставити мітку",
+	"ping-wheel.key.open-settings": "Відкрити налаштування",
+	"ping-wheel.key.name-labels": "Показувати ярлики імен",
+	"ping-wheel.subtitles.ping": "Мітка поставлена",
+
+	"ping-wheel.settings.pingVolume": "Гучність мітки: %s",
+	"ping-wheel.settings.pingDuration": "Тривалість мітки: %s",
+	"ping-wheel.settings.pingDistance": "Дистанція мітки: %s",
+	"ping-wheel.settings.correctionPeriod": "Період коригування: %s",
+	"ping-wheel.settings.itemIconVisible": "Піктограми предметів",
+	"ping-wheel.settings.directionIndicatorVisible": "Індикатор напрямку",
+	"ping-wheel.settings.nameLabelForced": "Завжди показувати імена",
+	"ping-wheel.settings.pingSize": "Розмір мітки: %s",
+	"ping-wheel.settings.channel": "Канал мітки",
+	"ping-wheel.settings.channel.tooltip": "Мітки один одного бачитимуть лише користувачі певного каналу",
+
+	"ping-wheel.value.hidden": "Приховано",
+	"ping-wheel.symbol.infinite": "∞",
+	"ping-wheel.unit.seconds": "%s сек",
+	"ping-wheel.unit.meters": "%s м",
+	"ping-wheel.unit.percent": "%s%%",
+
+	"ping-wheel.settings.title": "Налаштування «Ping Wheel»",
+	"ping-wheel.name": "Ping Wheel",
+
+	"ping-wheel.command.config.description": "відкрити інтерфейс налаштувань",
+	"ping-wheel.command.channel.get.description": "дізнатися ваш канал",
+	"ping-wheel.command.channel.set.description": "встановити канал, використовуйте «» для усталеного каналу",
```

Comment:
The syntax for the set command still uses `""`, since that is fixed by minecraft. So this should mention `\"\"` again like before